apix-integrationtests/build.gradle

```diff
@@ -118,27 +131,13 @@ subprojects {
         classpath = sourceSets.main.runtimeClasspath
 
         doFirst {
-            def serviceInfo = composeUpTask.servicesInfos.get("alfresco-core")
-            def port = serviceInfo.ports[8080]
-            def host = serviceInfo.getHost()
-            systemProperty("alfresco.url", "http://admin:admin@" + host + ":" + port + "/alfresco")
+            def endpoint = alfrescoDynamicExtensions.repository.endpoint
+            def alfresco_url = "${endpoint.protocol.get()}://admin:admin@${endpoint.host.get()}:${endpoint.port.get()}/alfresco"
+            println("Running integration tests on alfresco.url = ${alfresco_url}")
+            systemProperty("alfresco.url", alfresco_url)
         }
         outputs.upToDateWhen { false }
-
         // After the tests, the docker setup should be stopped
         finalizedBy(composeDownTask)
     }
-
-    task integrationTestLocal(type: Test) {
-        group 'verification'
-        dependsOn installTestsLocal
-        testClassesDirs = sourceSets.main.output.classesDirs
-        classpath = sourceSets.main.runtimeClasspath
-        def host = project.hasProperty('host') ? project.host : 'localhost'
-        def port = project.hasProperty('port') ? project.port : '8080'
-        def protocol = project.hasProperty('protocol') ? project.protocol : 'http'
-        systemProperty("alfresco.url", "$protocol://admin:admin@" + host + ":" + port + "/alfresco")
-
-        outputs.upToDateWhen { false }
-    }
 }
```

Comment:
Do you still have a task for just installing the latest impl jar in the running container?

Reply:
This only contains tasks to install integration test jars?
And yes, that's possible. It's added in the readme how to do it